spec/index/multi_index_spec.rb

```diff
@@ -35,6 +35,30 @@
           labels: [[0,0,1,1,2,2]])
       }.to raise_error
     end
+
+    context "create an MultiIndex with name" do
+      context "if no name is set" do
+        let(:mi) { Daru::MultiIndex.new(
+                    levels: [[:a,:b,:c], [:one, :two]],
+                    labels: [[0,0,1,1,2,2], [0,1,0,1,0,1]]) }
+        it { expect(mi.name).to be_nil }
+      end
+
+      context "correctly return the MultiIndex name" do
+        let(:mi) { Daru::MultiIndex.new(
+                  levels: [[:a,:b,:c], [:one, :two]],
+                  labels: [[0,0,1,1,2,2], [0,1,0,1,0,1]], name: ['n1', 'n2']) }
+        it { expect(mi.name).to eq ['n1', 'n2'] }
```

Comment:
Sorry @v0dro @zverok , I missed this comment. I will add testcases for this in next commit.

>Can more than one index level have the same name?

```
irb(main):007:0> mi.name = ['k1', 'k1']
=> ["k1", "k1"]
irb(main):008:0> mi
=> #<Daru::MultiIndex(6x2)>
  k1  k1
   a one
     two
   b one
     two
   c one
```

> Are they nil? 
In this PR we can't access one level name.
In table space is created if name is not set for index column.
```
irb(main):005:0> mi.name = ['k1']
=> ["k1"]
irb(main):006:0> mi
=> #<Daru::MultiIndex(6x2)>
      k1
   a one
     two
   b one
     two
   c one
     two
```